lib/commands/init.js

```diff
@@ -64,5 +64,16 @@ module.exports = function (path = '', local, theme) {
     replace(target(filename), 'repo: \'\'', `repo: '${repo}'`)
   }
 
+  if (plugins) {
+    replace(target(filename), '_plugins_', '_plugin\n  '.repeat(plugins.length))
+    plugins.forEach(plugin => {
+      const url = plugin.includes('//') ? plugin : `https://unpkg.com/docsify/lib/plugins/${plugin}.min.js`
```

Comment:
Can you change the cdn to `jsdelivr` also we need a version for the docsify. 
Ref : https://github.com/docsifyjs/docsify-cli/pull/107